site/rbac.md

```diff
@@ -1,24 +1,24 @@
-# RBAC
+# Role-Based Access Control (RBAC)
 
-The operator assumes that certain roles and role bindings are created on the Kubernetes cluster.  The operator installation scripts create these, and the operator verifies that they are correct when the cluster starts up.  This section lists the [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) definitions that are created.
+The operator assumes that certain roles and role bindings are created on the Kubernetes cluster.  The operator installation scripts create these, and the operator verifies that they are correct when the cluster starts up.  This document lists the [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) definitions that are created.
 
-The general design goal is to provide the operator with the minimum amount of permissions that it requires, and to favor built-in roles over custom roles where it make sense to do so.
+The general design goal is to provide the operator with the minimum amount of permissions that it requires, and to favor built-in roles over custom roles, where it make sense to do so.
 
-## KUBERNETES ROLE DEFINITIONS
+## Kubernetes role definitions
 
-| Cluster Role | Resources | Verbs | Notes |
+| Cluster role | Resources | Verbs | Notes |
 | --- | --- | --- | --- |
-| weblogic-operator-cluster-role |	namespaces, persistentvolumes	| get, list, watch | 1 |
+| `weblogic-operator-cluster-role` |	namespaces, persistentvolumes	| get, list, watch | 1 |
 | |	customresourcedefinitions in API group apiextensions.k8s.io	| get, list, watch, create, update, patch, delete, deletecollection	| |
 | |	domains in API group weblogic.oracle	| get, list, watch, update, patch	| |
 | |	Ingresses in API group extensions	| get, list, watch, create, update, patch, delete, deletecollection	| |
-| weblogic-operator-cluster-role-nonresource	| nonResourceURLs: ["/version/*"]	| get |	1 |
-|weblogic-operator-namespace-role	| secrets, persistentvolumeclaims	| get, list, watch	| 2 |
+| `weblogic-operator-cluster-role-nonresourc`e	| nonResourceURLs: ["/version/*"]	| get |	1 |
```

Comment:
typo I think you meant `weblogic-operator-cluster-role-nonresource`

Reply:
Yep, I'll fix it now.